app/models/user.rb

```diff
@@ -31,4 +31,20 @@ def is_reviewer?
   def is_applicant?
     return role == 0
   end
+
+  def first_name
+    if name.split.count > 1
+      name.split.first
+    else
+      name
+    end
+  end
+
+  def last_name
+    if name.split.count > 1
+      name.split.last
+    else
+      name
+    end
+  end
```

Comment:
Feel free to use these two helper methods `user.first_name` and `user.last_name`